plugin/Form.php

```diff
@@ -600,6 +601,26 @@ public function getEstateContextLabel(): string
 		return esc_html($result);
 	}
 
+	/**
+	 *
+	 * @param bool $isCaptchaRequired
+	 *
+	 */
+
+	private function renderCaptchaScript(bool $isCaptchaRequired = false)
+	{
+		$onofficeSettingsCaptchaSiteKey = get_option('onoffice-settings-captcha-sitekey', '');
+		if ($onofficeSettingsCaptchaSiteKey !== '') {
+			if (wp_script_is('onoffice-captchacontrol', 'registered')) {
+				echo '<script>var requiresCaptchaForm = ' . json_encode($isCaptchaRequired) . ';</script>';
+			} else {
+				wp_register_script('onoffice-captchacontrol', plugins_url('/js/onoffice-captchacontrol.js', ONOFFICE_PLUGIN_DIR . '/index.php'), 'google-recaptcha');
+				add_action('wp_footer', function () use ($isCaptchaRequired) {
+					echo '<script>var requiresCaptchaForm = ' . json_encode($isCaptchaRequired) . ';</script>';
+				});
+			}
+		}
+	}
```

Comment:
Maybe you dont need boolean requiresCaptchaForm, if you use CaptchaDataChecker(oder Handler) to load onoffice-captcha.js only if repatcha is enabled and we are on a form page.

See comment in CaptchaDataChecker.php